extensions/wikia/Blogs/Blogs.php

```diff
@@ -112,6 +112,9 @@ function wfBlogsOnBeforeInitialize( &$title, &$article, &$output, User $user, $r
 // Usages of images on blogs on file pages
 $wgHooks['FilePageImageUsageSingleLink'][] = 'BlogsHelper::onFilePageImageUsageSingleLink';
 
+// Checking that user is permitted to delete blog articles
+$wgHooks['BeforeDeletePermissionErrors'][] = 'BlogLockdown::onBeforeDeletePermissionErrors';
```

Comment:
It is literally impossible to do this because this method gets called multiple times for a single requests and with separate action types. That was my first possible solution to no avail.